src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -24,6 +24,19 @@ uct_rc_mlx5_ep_fence_put(uct_rc_mlx5_iface_common_t *iface, uct_ib_mlx5_txwq_t *
     uct_rc_ep_fence_put(&iface->super, &txwq->fi, rkey, addr, offset);
 }
 
+/**
+ * Pass the CQE zipping flag only if both
+ * runtime and compile time flags are set
+ */
+static UCS_F_ALWAYS_INLINE int
+uct_rc_mlx5_check_zip_flag(uct_rc_mlx5_iface_common_t *iface, int dir,
+                           int flags) {
+    if ((flags & UCT_IB_MLX5_POLL_FLAG_CQE_ZIP) && !iface->cq[dir].zip) {
```

Comment:
why need to check iface->cq[dir].zip?
we didn't do it before this PR

Reply:
Because of what we discussed previously. Before this patch we had only two variants of `progress` function: with and without zipping. And this options could be simply expressed by passing compile-time poll flags only.

Now we have four options:
- no zipping
- tx zipping
- rx zipping
- tx and rx zipping

Expression all these options by compile-time `flags` only will cause exponential growth of different `progress` functions. So, as I remember, we decided to implement mixed approach: existing of zipping still expressed by compile time flags, but rx/tx zipping presence checked using per CQ values.

Reply:
what if we just pass poll_flags as-is to both rx poll and tx poll?

Reply:
Seems like we can, but want to recheck the perf.